pkg/client/injection/reconciler/pipeline/v1beta1/pipelinerun/controller.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright 2019 The Tekton Authors
```

Comment:
Should the generated code have 2020 in here?

Reply:
I can change this and regenerate, but this is where that comes from: https://github.com/tektoncd/pipeline/blob/master/hack/boilerplate/boilerplate.go.txt#L2

Reply:
Oh, I see - time to update that indeed :)

Reply:
I pushed another commit with the timestamp changed and codegen rerun

Reply:
thanks!